examples/sqs/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+
+	"github.com/beatlabs/patron"
+	"github.com/beatlabs/patron/component/async"
+	patronsqs "github.com/beatlabs/patron/component/async/sqs"
+	"github.com/beatlabs/patron/log"
+)
+
+type sqsConfig struct {
+	endpoint string
+	name     string
+	region   string
+}
+
+// Override these values and make sure you have credentials and a session token in your ~/.aws/credentials file.
+// Then set the appropriate value to an AWS_PROFILE env var when running this example.
+// The assumed role needs to have access to the queue.
+var sampleConfig sqsConfig = sqsConfig{
+	endpoint: "https://sqs.eu-west-1.amazonaws.com/315266612443/sandbox-payin",
+	name:     "sandbox-payin",
+	region:   "eu-west-1",
+}
+
+func init() {
+	err := os.Setenv("PATRON_LOG_LEVEL", "debug")
+	if err != nil {
+		fmt.Printf("failed to set log level env var: %v", err)
+		os.Exit(1)
+	}
+}
+
+func main() {
+	name := "sqs"
```

Comment:
Maybe we could also join both of them into the one example in order to avoid having a lot of them. The fifth example could be reused in order to showcase this feature.

Reply:
@mantzas Isn't better to have several smaller examples instead of fewer large examples?

Reply:
Yeah I don't like mixing stuff like this. The fifth example greets another running service when consuming the message. I'd like examples to be completely standalone.

Reply:
the idea is to change the fifth to be supporting this feature only. We can leave it as is no problem.